LinqToDBPostGisNetTopologySuite/GeometryAccessors.cs

```diff
@@ -311,7 +309,7 @@ public static string STGeometryType(this NTSG geometry)
         /// </remarks>
         /// <param name="geometry">Input geometry.</param>
         /// <returns>Geometry type string (with 'ST_' prefix).</returns>
-        [Sql.Function("ST_GeometryType", ServerSideOnly = true)]
+        [Sql.Expression("ST_GeometryType({0}::geometry)", ServerSideOnly = true)]
```

Comment:
Could be a better approach than 9fdcaae541273f7155fc51dc0292fb298fc11188 workaround with version check... will it work in PostGIS 2.5 ?

Reply:
Works on my machine.